### ISSUE TITLE
1. XML auschecken

### ISSUE THREAD

**Issue by cutterkom** [queersearch/queersearch-data-transformations#1]

**Comment by cutterkom** [comment]
https://github.com/queersearch/queersearch-data-transformations/blob/main/code/ffbiz/extract-xml-radusch.R

**Comment by cutterkom** [comment]
Ich hab die extrahiertem Daten mal hier in einem Sheet gespeichert: https://docs.google.com/spreadsheets/d/1Bqv2mqZiLA8KgppsxZXEmQj3ty89BHyLQCnGKWX_NU4/edit?gid=0#gid=0

Ist das etwas, womit wir aus euren Perspektiven weiterarbeiten könnten? @janseix , @esra-paul

**Comment by janseix** [comment]
auf den ersten Blick sieht das doch sehr gut aus

**Comment by janseix** [comment]
auf den zweiten Blick fallen mir dann ein paar Sachen auf. Habe mal das Tabellenblatt dubliziert und ein paar Hervorhebungen gemacht.

- an einigen Stellen wird <p> angezeigt anstatt eines Leerzeichens oder Absatzes?
- Spalte M und Q: beides Spalten für Enthältvermerk?
- sind Spalte A und C identisch?


ich denke, wir sollte vermutlich auch irgendwie dokumentieren, welche Spalten in den Ursprungsdaten sind und welche wir hinzufügen, oder?

**Comment by cutterkom** [comment]
Danke für's genaue ansehen!

Ja, A und C sind gleich. Die musste ich beim ETL rausziehen, weil... ach egal :) 

ja, M und Q erheben sich daraus, dass da halt anstatt gscheid zu modellieren einfach ein neues Feld gemacht wurde, siehe Orginal: 
<img width="1153" alt="grafik" src="https://github.com/user-attachments/assets/447780a7-b626-43ac-8529-ae6d2ab6862e">


den ersten Bulletpoint versteh ich nicht. Meinst du `<p>` anstatt Absatz? Sowas zB wäre ein Cleaning-Schritt, der genausow wie whitespaces weg einmal an einer Stelle gemacht werden muss. Evtl können wir's aber auch einfach genauso lassen, weil die `<p></p>`'s ja in Omeka S genauso dargestellt werden

**Comment by janseix** [comment]
genau, ich meine <p> das p in Klammern, das wurde mir hier wohl auch schon einfach in einen Absatz umgewandelt. Damit hat sich dann meine Frage geklärt, ob in den Daten dort ein Leerzeichen oder ein Absatz ist/sein sollte.

**Comment by esra-paul** [comment]
ich finde auch, das sieht gut aus! in Omeka S würde man vermutlich eine Modellierung für Nachlass --> einzelne Ordner/Bestandteile und eine für Nachlass --> Gesamtbestand erstellen, um das auch hierarchisch abbilden zu können.
